.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions and Answers
+    url: https://github.com/seqan/sharg-parser/discussions
```

Comment:
Do we check those? I thought people should just open issues?

Reply:
It feels like the discussions feature is designed to serve as a Q/A platform, so we should try and use it.

I.e. 
bugs -> issue
question -> discussion

Reply:
sure we can try.